students/Jaidjen/lesson04/Mailroom2.py

```diff
@@ -0,0 +1,77 @@
+import sys  # imports go at the top of the file
+
+
+Mail = {1:{'Name':'James Smith', 'Donation':[2000.25,500,27.50]},
+              2:{'Name': 'Dorothy Parker','Donation':[3175,475.45,92.30]},
+              3:{'Name':'Dohgyu Hwang','Donation':[10670,350.19,175]},
+              4:{'Name': 'Sally Yu', 'Donation':[100,925]},
+              5:{'Name':'Jenny Ramos','Donation':[25.14]}}
```

Comment:
what is the benefit of using numbers for keys? the benefit of dictionaries is cheap look up by key with a natural key - your natural key here is the donor name.